data/download.yaml

```diff
@@ -0,0 +1,42 @@
+
+binaries:
+  platforms:
+  - darwin
+  - linux
+  - windows
+  platformInfo:
+    darwin:
+      name: macOS
+      icon: fa-apple
+    linux:
+      name: Linux
+      icon: fa-linux
+    windows:
+      name: Windows
+      icon: fa-windows
+  fullReleaseVersions:
```

Comment:
the buttons look great, but I am reluctant to start listing all versions, including minor patches. That sounds like a lot of maintenance work for little value since the versions can easily be seen from the actual download pages (e.g. GitHub Releases). There's also ambiguity as we are changing which binaries are being included in the archives, e.g. HotROD was not built as executable previously.

Reply:
Related to the comment above, perhaps we can merge this but make the download page a draft, then you can adjust the versions, images, etc. and then remove the draft status of the page?